dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -552,13 +557,39 @@ export default defineComponent({
         this.postEuTaxonomyDataForFinancialsProcessed = false;
         this.messageCount++;
 
+        // JSON.parse/stringify used to clone the formInputsModel in order to stop Proxy refreneces
+        const clonedFormInputsModel = JSON.parse(JSON.stringify(this.formInputsModel)) as ObjectType;
```

Comment:
The problem is that the object has Proxies inside it, which  means that attempts at using shallow cloning like `{ ...obj }` or `Object.assign( {}, obj )` , meant that those Proxies were also inherited (as per nature and intention of a Proxy).
Unfortunately, `structuredClone` cannot be used either since the Proxies inside the object are somehow connected with the Window object, and attempting to do so, throws the following error: `Failed to execute 'structuredClone' on 'Window': #<Object> could not be cloned.`

By using `JSON.parse(JSON.stringify( obj ))` we break the connection to the Proxy instance and receive a clean, new object with just the data needed for the request payload.

It's an older code, but it checks out.

Reply:
All right. Thanks for the explanation :)